h/static/scripts/test/bridge-test.coffee

```diff
@@ -34,21 +33,19 @@ describe 'Bridge', ->
   describe '.createChannel', ->
     it 'creates a new channel with the provided options', ->
       Channel.build.returns(createChannel())
-      bridge = createBridge()
       bridge.createChannel('WINDOW', 'ORIGIN', 'TOKEN')
 
       assert.called(Channel.build)
       assert.calledWith(Channel.build, {
         window: 'WINDOW'
         origin: 'ORIGIN'
-        scope: 'bridge:TOKEN'
+        scope: 'annotator:bridge:TOKEN'
```

Comment:
I think you can remove this change and we can just stop prefixing the scope with `annotator:bridge` since that serves no purpose.